services/order.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order, MovieSession
+
+
+def create_order(
+        tickets: list[dict],

```

Comment:
The type hint `list[dict]` is only supported in Python 3.9 and later. If you need compatibility with earlier versions, consider using `List[Dict]` from the `typing` module.